src/lib/utils/featureFlag.ts

```diff
@@ -0,0 +1,24 @@
+export const featureFlags: string[] = [];
+
+export const isFeatureFlagEnabled = (flag: string) => {
+  if (!featureFlags.includes(flag)) {
+    return false;
+  }
+  const value = localStorage.getItem(flag);
+  if (value === null || value === "false") {
+    return false;
+  }
+  return true;
+};
+
+export const setDefaultFlags = () => {
+  featureFlags.forEach((f) => localStorage.setItem(f, "false"));
+};
+
+export const setFeatureFlag = (flag: string, value: boolean) => {
+  if (!featureFlags.includes(flag)) {
+    return false;
+  }
+  localStorage.setItem(flag, value.toString());
```

Comment:
Maybe instead of having many different storage entries for each feature flag, perhaps we can wrap it in an object and JSON serialize/parse it? But this is opinionated and skip it if it's too big of a hassle

Reply:
I think it's worth keeping as is, since this way it's easier to enable/disable flags manually through dev tools if needed